www/source/blog/2018-05-11-changes-in-the-0.56.0-supervisor.html.md

```diff
@@ -0,0 +1,74 @@
+
+`hab svc stop` is how you stop a currently-running (and thus, loaded) service. A stopped service will not be restarted if you restart the Supervisor; you must run `hab svc start`.
+
+`hab sup run` is now the _only_ way to start a Supervisor. As a convenience, `hab sup run` can also accept the same arguments as `hab svc load` in order to start a supervisor, and load and start a single service (for example, `hab sup run core/redis --strategy=at-once --group=dev` --peer=192.168.1.1:9638`). This means that you are still able to start a Supervisor and service in a single command.
+
+`hab sup term` shuts down the Supervisor and all currently-running services.
+
+Another frequently-asked question centered around the difference between "persistent" and "transient" services, which happened to also be tied into the "start" vs. "load" question. This was also an area with lots of subtle distinctions which added to the confusion. Now, there is no longer any distinction between persistent and transient services; there are just services. The details of when they start and when they restart are completely determined by the behaviors of the above commands.
+
+## Authentication of Communication
+
+Now that Supervisor interaction is accomplished over TCP, rather than direct filesystem manipulation, we need to add some additional layers of safety.
+
+First, the Supervisor listens on the `127.0.0.1` interface (port 9632) by default, so without any additional configuration, you will only be able to interact with a Supervisor from the same host (just as with older Supervisors). If you wish to open this up to allow remote control of your Supervisors, you'll want to start your Supervisor with the `--listen-ctl` option set to a different interface, e.g. `--listen-ctl=0.0.0.0:9632`.
+
+The Supervisor communication protocol is also protected by a shared secret. When starting up, the Supervisor will look for the `/hab/sup/default/CTL_SECRET` file, which contains the secret it will use to authenticate any command messages it receives. If this file does not exist at startup, the Supervisor will generate a secret itself and write it to that location. If you only interact with your Supervisor from the same host, this will not be an issue, as the client can ultimately use this same secret transparently. However, if you _do_ wish to work with your Supervisors remotely, you'll need to manage your secrets a bit.
+
+There is a new `hab sup secret generate` command that will generate a random secret (on standard output) that you can use. Placing this in the `/hab/sup/default/CTL_SECRET` file for your Supervisor(s) will take care of one half of the interaction. The `hab` client, on the other hand, can either use a secret placed in the `HAB_CTL_SECRET` environment variable, or in your `~/.hab/etc/cli.toml` file, under the `ctl_secret` key.
```

Comment:
This can also be set by running `hab setup` - not sure if we care to message that. Personally I like configuring by the config file